### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Dani Tiago Faria dos Santos and Nicolas Duprat

### DISCUSSION THREAD

**Discussion by Dansnts** [heig-vd-dai-course/heig-vd-dai-course#332] [Show and tell]
Create a program who can encrypt and decrypt text and binary files with picoCLI.

Convert from and to emoji, Cesar Encrypt, Number Encrypt, etc ...

**Comment by ludelafo** [comment]
Seems good to us!

**Comment by Dansnts** [comment]
Project is over !

https://github.com/Dansnts/DAI-Labo01

Hash : [a3bc3b4acba33e49851755df0631c9bc49b7e543](https://github.com/Dansnts/DAI-Labo01/commit/c018a062d6c22eb2615b568f5513f3bbe5a67833)

(We also have a 1.0 release)

**Reply by Dansnts** [reply]
If by any chance you can't access the git please let me know !

**Reply by ludelafo** [reply]
Hello @Dansnts, @skyrag,

Could you add @hadrylouis to your repository as well to grade your work?

Thank you!

**Reply by Dansnts** [reply]
Hello @ludelafo,

Sorry, we just saw the message today. Hope it didn't cause any trouble. We'll also add @hadrylouis to our future repositories from now on.

Sorry for the inconvenience!
